src/components/PersonaCard/PersonaCard.md

```diff
@@ -0,0 +1,37 @@
+# Persona Card
+Includes a Persona component with contact options.
```

Comment:
I would be a bit more verbose here to help explain usage:
"The visualization of details of an individual including Skype contact details, email, location information, and location within the hierarchy of an organization. Within Office 365, PersonaCards often appear when a user hovers, taps, or clicks on a Persona."